libmscore/layout.cpp

```diff
@@ -3508,6 +3508,9 @@ void Score::layoutSystemElements(System* system, LayoutContext& lc)
                   if (!mb->isMeasure())
                         continue;
                   Measure* m = toMeasure(mb);
+                  if (MeasureNumber* mno = m->noText(staffIdx))
+                        ss->skyline().add(mno->bbox().translated(m->pos() + mno->pos()));
```

Comment:
FWIW, could be worth also adding a check for mno->autoplace() before adding to the skyline.  It's one of the few such things I didn't do in my PR #4730.  I figure I'll need to update that anyhow if this gets merged first, though.